ultimatejobweb/views.py

```diff
@@ -1,6 +1,19 @@
 from django.shortcuts import render
+from django.contrib.auth import authenticate
+from django.contrib.auth.forms import UserCreationForm
+from django.shortcuts import redirect
 
 
 def homepage(request):
     return render(request, r"web/ultimate_job_homepage.html")
 # Create your views here.
+
+
+def signup_view(request):
+    form = UserCreationForm(request.POST)
+    if form.is_valid():
+        form.save()
+        username = form.cleaned_data.get('username')
+        password = form.cleaned_data.get('password1')
+        return redirect('homepage')
```

Comment:
The user is created when you call form.save()
What are you doing with the username and password variables?

Reply:
> The user is created when you call form.save()
> What are you doing with the username and password variables?

I assume, If he would like to login the user and redirect it to home page
user = authenticate(username=username, password=password)
            login(request, user)
@BarBenNaim ^^